androidapp/HopSquadApp/app/src/main/java/com/hopsquad/hopsquadapp/activities/BaseActivity.java

```diff
@@ -0,0 +1,12 @@
+package com.hopsquad.hopsquadapp.activities;
+
+import android.support.v7.app.AppCompatActivity;
+
+/**
+ * Created by memo on 15/11/17.
+ */
+
+public class BaseActivity extends AppCompatActivity {
```

Comment:
Was this move just for organization purposes or are you thinking they'll eventually share more things?

Reply:
Both, I was having some trouble with fragments and activities versions and I had to change the base Activity class a bunch of times, so instead added this one and only changed it once for all activities.